pkg/apis/core/v1/conversion.go

```diff
@@ -270,6 +274,117 @@ func Convert_v1_PodTemplateSpec_To_core_PodTemplateSpec(in *v1.PodTemplateSpec,
 	return nil
 }
 
+// helper for v1.PodIPInfo list to core.PodIPinfoList
+func v1_PodIPs_To_core_PodIPs(inList []v1.PodIPInfo) []core.PodIPInfo {
+	out := make([]core.PodIPInfo, len(inList))
+	for idx, value := range inList {
+		out[idx] = core.PodIPInfo{
+			IP:         value.IP,
+			Properties: value.Properties,
+		}
+	}
+	return out
+}
+
+// Converts podStatus from v1 to core
+// auto + handling podIp/podIps
+func Convert_v1_PodStatus_To_core_PodStatus(in *v1.PodStatus, out *core.PodStatus, s conversion.Scope) error {
+	if err := autoConvert_v1_PodStatus_To_core_PodStatus(in, out, s); err != nil {
+		return err
+	}
+
+	// handle ip/ips fields
+	if 0 < len(in.PodIP) && 0 < len(in.PodIPs) {
```

Comment:
this is a bit hard to read... `if len(in.PodIP) > 0 && len(in.PodIPs) > 0 {` is more conventional...

Reply:
this is now fixed